tests/conftest.py

```diff
@@ -183,3 +184,12 @@ def metadata_sign_input() -> List[str]:
     ]
 
     return input
+
+
+@pytest.fixture()
+def mocked_os_makedirs(monkeypatch):
```

Comment:
Given it's used only for the `repository_service_tuf.cli.artifact.download.os.makedirs`, isn't it better to be in the corresponding `test_download.py`?